strategy.html

```diff
@@ -0,0 +1,60 @@
+---
+Title: Events
+layout: default
+---
+
+<div class="pure-g">
+  <div class="pure-u-1">
+    <h1>Strategic Vision</h1>
+
+    <p>This is a strategic vision for Ruby NZ for 2019-2020. We will use this as a guide for the actions we take. </p>
+
+    <p>Our charter specifies the following purposes:</p>
+    <ul>
+    <li>Foster and support the Ruby programming language, its users, community and ecosystem in New Zealand;</li>
```

Comment:
```suggestion
    <li>Foster and support the Ruby programming language, its users, community and ecosystem in New Zealand.</li>
```

the following items use full-stops, so should they all use full-stops?